raiden/tests/integration/network/proxies/test_service_registry.py

```diff
@@ -39,7 +39,8 @@ def test_service_registry_random_pfs(
 
     # Test that get_service_address by index works
     for idx, address in enumerate(addresses):
-        c1_service_proxy.get_service_address('latest', idx) == address
+        assert c1_service_proxy.get_service_address('latest', idx) == address
```

Comment:
This was the same kind of bug introduced by another PR, which would have been caught by the tool